infra/resources/modules/commons/function_consumptions.tf

```diff
@@ -236,8 +236,8 @@ module "function_consumptions_autoscaler" {
 
   scheduler = {
     normal_load = {
-      default = 3
-      minimum = 3
+      default = 1
+      minimum = 1
```

Comment:
**note**: The consumer function processes jobs asynchronously, so at a minimum, it can have only one instance.

Reply:
It has multi az enabled, so it can't go below 3 instances